Shared/Samples/Project with chosen transformation/ProjectWithChosenTransformationView.swift

```diff
@@ -0,0 +1,145 @@
+                    model.pointGraphic.geometry = specificProjectedPoint
+                    
+                    // Updates the location callout placement.
+                    calloutPlacement = CalloutPlacement.location(specificProjectedPoint)
+                } else {
+                    // Hides the callout and point graphic.
+                    calloutPlacement = nil
+                    model.pointGraphic.geometry = nil
+                }
+            }
+            .callout(placement: $calloutPlacement.animation(.default.speed(2))) { _ in
+                VStack(alignment: .leading) {
+                    Group {
+                        Text("Coordinates")
+                            .fontWeight(.medium)
+                        Text("Original (WGS 84):\n\(originalPoint.xyCoordinates)")
+                        Text("Projected (non-specific):\n\(unspecifiedProjectedPoint.xyCoordinates)")
+                        Text("Projected (WKID: 108055):\n\(specificProjectedPoint.xyCoordinates)")
+                    }
+                    .font(.callout)
+                }
+                .padding(6)
+            }
+            .overlay(alignment: .top) {
+                Text("Tap on the map.")
+                    .frame(maxWidth: .infinity)
+                    .padding(.vertical, 6)
+                    .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+            }
+    }
+}
+
+private extension ProjectWithChosenTransformationView {
+    /// The model used to store the geo model and other expensive objects
+    /// used in this view.
+    final class Model {
+        /// A map with a WGS84 world basemap and an initial viewpoint over Mongolia.
+        let map: Map = {
+            let portalItem = PortalItem(
+                portal: .arcGISOnline(connection: .anonymous),
+                id: .worldBasemaps
+            )
+            let map = Map(item: portalItem)
+            
+            // The transformation used in this sample is only valid in the following area.
+            let extent = Envelope(
+                xRange: 87.76...119.94007,
+                yRange: 41.58...52.1500,
+                spatialReference: .wgs84
+            )
+            map.initialViewpoint = Viewpoint(boundingGeometry: extent)
+            return map
+        }()
+        
+        /// A graphics overlay containing a graphic with a circular, red marker symbol.
+        let graphicsOverlay = GraphicsOverlay(graphics: [
+            Graphic(symbol: SimpleMarkerSymbol(color: .red, size: 8))
+        ])
+        
+        /// The graphic with a circular, red marker symbol.
+        var pointGraphic: Graphic { graphicsOverlay.graphics.first! }
+    }
+}
+
+private extension FormatStyle where Self == FloatingPointFormatStyle<Double> {
+    /// Formats the double with four decimals places of precision.
+    static var decimal: Self {
+        Self.number.precision(.fractionLength(4))
```

Comment:
```suggestion
        number.precision(.fractionLength(4))
```

I think `Self` is implied here.